www/src/templates/template-blog-post.js

```diff
@@ -116,9 +116,6 @@ class BlogPostTemplate extends React.Component {
                     mt: 3,
                     mb: 9,
                   },
-                  [mediaQueries.lg]: {
-                    ml: `-8em`,
```

Comment:
Where did this originate, out of curiosity?

Reply:
It wasn't in the previous commit. I wonder if it got introduced in a weird merge or if @fk was working on layouts and didn't notice it at the large screen size while reworking that page with the sx prop. I didn't poke too much through the commit log but in the history for the file it wasn't in the previous commit but was in the latest adding dark mode.